shardingsphere-infra/shardingsphere-infra-optimize/src/main/java/org/apache/shardingsphere/infra/optimize/core/metadata/FederateTableMetadata.java

```diff
@@ -88,4 +96,13 @@ private DataSource getActualDataSource(final Map<String, DataSource> dataSources
         }
         return dataSources.get(result);
     }
+    
+    /**
+     * Get column names.
+     * 
+     * @return column names collection
+     */
+    public List<String> getColumnNames() {
```

Comment:
Maybe a @Getter is another better option?

Reply:
> Maybe a @getter is another better option?

I will optimize it.